apps/mosaico/web/index.js

```diff
@@ -17,6 +17,7 @@ function main() {
 	mostReadArticles={window.mostReadArticles || null}
 	frontpageArticles={window.frontpageArticles || null}
   staticPageContent={window.staticPageContent || null}
+	tagListArticles={window.tagListArticles || null}
```

Comment:
We also need the purescript implementation. These are the `JSProp` of `Mosaico`, and we need to somehow convert that into `Props`. Then, we probably should put the `props.tagListArticles` into the initial state and render them if they exists.